test/libsolidity/SolidityNatspecJSON.cpp

```diff
@@ -204,6 +204,112 @@ BOOST_AUTO_TEST_CASE(dev_and_user_no_doc)
 	checkNatspec(sourceCode, "test", userNatspec, true);
 }
 
+BOOST_AUTO_TEST_CASE(dev_public_state_variable)
+{
+	char const* sourceCode = R"(
+		contract test {
+			/// @notice example of notice
+			/// @dev example of dev
+			uint public state;
+		}
+	)";
+
+	char const* natspec = "{\n"
+						  "    \"methods\" : \n"
+						  "    {\n"
+						  "        \"state()\" : \n"
+						  "        {\n"
+						  "            \"details\" : \"example of dev\"\n"
```

Comment:
Where is the @notice?

I think the `dev` documentation should only be added to `stateVariables` and not to the external interface.